tests/static/e2e/pages/report-case-page.ts

```diff
@@ -0,0 +1,101 @@
+import { expect, Locator, Page } from "@playwright/test"
+import { orgSlug, Routes } from "../routes"
+
+export class ReportCasePage {
+  readonly page: Page
+  readonly route: string
+  readonly reportHeader: Locator
+  readonly titleTextBox: Locator
+  readonly descriptionTextBox: Locator
+  readonly projectDropdown: Locator
+  readonly typeDropdown: Locator
+  readonly priorityDropdown: Locator
+  readonly submitButton: Locator
+  readonly pageBorder: Locator
+
+  constructor(page: Page) {
+    this.page = page
+    this.route = orgSlug + Routes.ReportCase
+    this.reportHeader = page.getByText("Open a Case").first()
+    this.titleTextBox = page.getByLabel("Title", { exact: true })
+    this.descriptionTextBox = page.getByLabel("Description", { exact: true })
+    this.projectDropdown = page.getByRole("combobox").filter({ hasText: "Project" })
+    this.typeDropdown = page.getByRole("combobox").filter({ hasText: "Type" })
+    this.priorityDropdown = page.getByRole("combobox").filter({ hasText: "Priority" })
+    this.submitButton = page.getByRole("button", { name: "Submit" })
+    this.pageBorder = this.page.locator("span").filter({
+      hasText: "Open a Case Cases are meant to triage events",
```

Comment:
This text needs review.

Reply:
Simplified string in test case to not include title